modules/installation-initializing.adoc

```diff
@@ -326,6 +326,15 @@ After you create the installation configuration file, you can modify the file to
 ====
 
 ... Select the default vCenter datastore to use.
++
+[WARNING]
+====
+You can specify the path of any datastore that exists in a datastore cluster. Before configuring this datastore for your cluster, disable storage Distributed Resource Scheduler (DRS) for the datastore cluster. By not disabling DRS, your datastore cluster might incur data loss, and the vSphere Problem Detector Operator emits alerts about your cluster existing on a datastore that exists in a datastore cluster.
```

Comment:
Add a link to https://docs.openshift.com/container-platform/4.13/installing/installing_vsphere/installing-vsphere.html#anti-affinity-vsphere_installing-vsphere ? Not sure if this section is the correct way to disable DRS for a DS cluster?

Reply:
Also should I provide a link to the vSphere Problem Detector Operator? 

https://docs.openshift.com/container-platform/4.13/installing/installing_vsphere/using-vsphere-problem-detector-operator.html#vsphere-problem-detector-viewing-events_vsphere-problem-detector